website/src/css/_navbar.scss

```diff
@@ -22,3 +22,47 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Fix navbar logo */
+.navbar__logo {
+  height: 26px;
+  margin: 0 10px 0 8px;
+}
+
+/* Restyle gamburger button */
+
+.navbar {
+  position: relative;
+  height: 70px;
+}
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  margin-right: 0;
```

Comment:
I don't think that `margin-right` will anyhow work if the position is absolute.

Reply:
With
![image](https://user-images.githubusercontent.com/103461364/218177367-8b9098a6-c99f-435a-8245-1f6a6908286b.png)
Without
![image](https://user-images.githubusercontent.com/103461364/218177973-ba0abf78-93e3-4188-841c-a4f401e5a5a6.png)

The "margin-right: 0.5rem" property comes into play and this, I suspect, is due to "display: inherit" which inherits from "flex".

Reply:
Hmm, that can be true. 🤔  Maybe you should just reset `margin: 0` as simple as it is? I mean you won't be happy if some other CSS rule adds margin to any of the side as it can affect your positioning.